projects/ox-ruby/project.yaml

```diff
@@ -0,0 +1,12 @@
+homepage: "https://github.com/ohler55/ox"
+language: ruby
+primary_contact: "TODO@template.test"
```

Comment:
Why not add your own contact info?

Reply:
Neither @mschwager nor I are contributors to the project. We used it as a demonstration of Ruzzy's easiness of use and PoC, not with a goal to upstream it. That said, having an example project might be beneficial, so removing it from the Pull Request may not be ideal. However, managing those reports isn't something we aim to undertake. What's the best course of action here?

Reply:
No worries!